### PR TITLE
show number of passes instead of 0 after sending transaction

### PR DESCRIPTION
**Pull request by jonathanraes**
not tested, will test at home with other phone.

### CODE REVIEW

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/ResultActivity.java [hunk 1]
```diff
@@ -64,7 +66,7 @@ public void onConfidenceChanged(TransactionConfidence transactionConfidence, Tra
                     @Override
                     public void run() {
                         setAuthorizationStatus(CONFIRMED);
-                        ((TextView) findViewById(R.id.voting_pass_amount)).setText("0");
+                        textVoterName.setText(getString(R.string.please_hand, preamble));
                         textVotingPasses.setText(R.string.voting_passes);
```
Comment (wkmeijer) [inline app/src/main/java/com/digitalvotingpass/digitalvotingpass/ResultActivity.java]: Set this text to 'ballot papers'

**Review by wkmeijer — CHANGES_REQUESTED** [review]
Snackbar with two lines flows under the card, margin for the card could be increased.
It is not very clear that 'has right to' changed to 'please hand out' in the card view. Maybe the look of that sentence could also be changed.

**Comment by jonathanraes** [comment]
wait for #139 and update translations

**Comment by jonathanraes** [comment]
processed comments, please check if the card view now has enough margin for 2 line snackbar

**Review by wkmeijer — APPROVED** [review]

**Pull request merged**